src/xmlgen.cpp

```diff
@@ -904,6 +904,10 @@ static void generateXMLForMember(const MemberDef *md,TextStream &ti,TextStream &
     if (md->isMutable()) t << "yes"; else t << "no";
     t << "\"";
 
+    t << " thread_local=\"";
```

Comment:
Since `thread_local` is rarely used, maybe better to only write something if it is `yes` like is done for the attributes below. That way, none of the examples should have to be updated and the size of the XML output doesn't increase with a lot of default text.

Reply:
Valid point, I'll adjust the proposed PR.

Wouldn't this be valid for `mutable` as well?

Reply:
> Wouldn't this be valid for mutable as well?

Yes, I guess so, but maybe some existing tools that parse the XML by now depend on it being always present, so probably safer to keep it as-is.